clients/src/main/java/org/apache/kafka/clients/producer/internals/RecordAccumulator.java

```diff
@@ -478,7 +482,7 @@ public void abortIncompleteBatches() {
      */
     private void abortBatches() {
         for (RecordBatch batch : incomplete.all()) {
-            Deque<RecordBatch> dq = dequeFor(batch.topicPartition);
+            Deque<RecordBatch> dq = getDeque(batch.topicPartition);
```

Comment:
Also, while you're updating this file, can you please remove the unused `cluster` parameter in `abortExpiredBatches`?